api-report/map.api.md

```diff
@@ -135,6 +135,52 @@ export interface ILocalValue {
     readonly value: any;
 }
 
+// @public (undocumented)
```

Comment:
We should avoid exposing these interfaces from the package, as they are internal implementation details.

Reply:
At least for the operation formats